zenity-setup.sh

```diff
@@ -656,7 +665,7 @@ else
 	setSetting doSetupMAME true
 	setSetting doSetupCemu true
 	setSetting doSetupXenia false
-
+	setSetting doSetupXenia true
```

Comment:
should probably keep this false

Reply:
It was supposed to be doSetupRMG, thank you for catching that!